README.md

```diff
@@ -55,19 +69,13 @@ def augment_data(images, labels):
   inputs = {"images": images, "labels": labels}
   outputs = augmenter(inputs)
   return outputs['images'], outputs['labels']
-```
-
-Augment a `tf.data.Dataset`:
 
-```python
+# Augment a `tf.data.Dataset`
 dataset = tfds.load('rock_paper_scissors', as_supervised=True, split='train')
 dataset = dataset.batch(64)
 dataset = dataset.map(augment_data, num_parallel_calls=tf.data.AUTOTUNE)
```

Comment:
It's probably worth adding a `.prefetch(tf.data.AUTOTUNE)` to this dataset.

afaik this is pretty much always a worthwhile step to help optimize training speed, so it seems like something we should include in a quickstart.